packages/rke2-coredns/rke2-coredns.patch

```diff
@@ -314,24 +467,68 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rke2-coredns/charts-original/templates
 +  {{ else }}
 +  clusterIP: {{ (lookup "v1" "ConfigMap" "kube-system" "cluster-dns").data.clusterDNS }}
    {{- end }}
-   {{- if .Values.service.externalTrafficPolicy }}
-   externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
+   {{- if .Values.service.externalIPs }}
+   externalIPs:
+diff -x '*.tgz' -x '*.lock' -uNr packages/rke2-coredns/charts-original/templates/service.yaml.orig packages/rke2-coredns/charts/templates/service.yaml.orig
+--- packages/rke2-coredns/charts-original/templates/service.yaml.orig
++++ packages/rke2-coredns/charts/templates/service.yaml.orig
+@@ -0,0 +1,42 @@
++apiVersion: v1
++kind: Service
++metadata:
++  name: {{ template "coredns.fullname" . }}
++  labels:
++    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
++    app.kubernetes.io/instance: {{ .Release.Name | quote }}
++    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
++    {{- if .Values.isClusterService }}
++    k8s-app: {{ .Chart.Name | quote }}
++    kubernetes.io/cluster-service: "true"
++    kubernetes.io/name: "CoreDNS"
++    {{- end }}
++    app.kubernetes.io/name: {{ template "coredns.name" . }}
++{{- if .Values.customLabels }}
++{{ toYaml .Values.customLabels | indent 4 }}
++{{- end }}
++  annotations:
++{{ toYaml .Values.service.annotations | indent 4 }}
++spec:
++  selector:
++    app.kubernetes.io/instance: {{ .Release.Name | quote }}
++    {{- if .Values.isClusterService }}
++    k8s-app: {{ .Chart.Name | quote }}
++    {{- end }}
++    app.kubernetes.io/name: {{ template "coredns.name" . }}
++  {{- if .Values.service.clusterIP }}
++  clusterIP: {{ .Values.service.clusterIP }}
++  {{- end }}
++  {{- if .Values.service.externalIPs }}
++  externalIPs:
++  {{ toYaml .Values.service.externalIPs | indent 4 }}
++  {{- end }}
++  {{- if .Values.service.externalTrafficPolicy }}
++  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
++  {{- end }}
++  {{- if .Values.service.loadBalancerIP }}
++  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
++  {{- end }}
++  ports:
++{{ include "coredns.servicePorts" . | indent 2 -}}
++  type: {{ default "ClusterIP" .Values.serviceType }}
 diff -x '*.tgz' -x '*.lock' -uNr packages/rke2-coredns/charts-original/values.yaml packages/rke2-coredns/charts/values.yaml
 --- packages/rke2-coredns/charts-original/values.yaml
 +++ packages/rke2-coredns/charts/values.yaml
-@@ -3,8 +3,8 @@
+@@ -3,7 +3,7 @@
  # Declare variables to be passed into your templates.
  
  image:
 -  repository: coredns/coredns
--  tag: "1.6.9"
 +  repository: rancher/hardened-coredns
-+  tag: "v1.6.9"
+   tag: "1.7.1"
```

Comment:
The actual tag here seems to be `v1.7.1`
https://hub.docker.com/layers/rancher/hardened-coredns/v1.7.1/images/sha256-2f943b97ae07055f177f9c89a429bf8a2b0afd85a4f85abc51def84773fef814
```suggestion
   tag: "v1.7.1"
```

Reply:
Yes, but, https://hub.docker.com/layers/rancher/hardened-coredns/v1.6.9/images/sha256-97f077ce82448672f87a98b54a8da5ebd46ca01aeb4be6e59990399b470a0865?context=explore

Reply:
Oh, I see what i did. :bonk: